airflow/api/common/experimental/mark_tasks.py

```diff
@@ -70,7 +71,7 @@ def set_state(
     past: bool = False,
     state: TaskInstanceState = TaskInstanceState.SUCCESS,
     commit: bool = False,
-    session=None,
+    session: SASession = None,
```

Comment:
Please search for `session: Session = NEW_SESSION` for a more idomatic way to annotate this.

Reply:
Thanks